Plugins/Events/Events.cpp

```diff
@@ -89,6 +90,11 @@ Events::Events(const Plugin::CreateParams& params)
         m_itemEvents = std::make_unique<ItemEvents>(GetServices()->m_hooks);
     }
 
+    if (GetServices()->m_config->Get<bool>("ENABLE_ITEM_EVENTS", true))
```

Comment:
I guess this should be ENABLE_FEAT_EVENTS?

Reply:
Good catch. Fix'd